app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb

```diff
@@ -92,14 +92,16 @@ def parse_host_guest_devices(hardware, sys)
   end
 
   def parse_host_advanced_settings(host, sys)
-    persister.hosts_advanced_settings.build(
-      :resource     => host,
-      :name         => "pcm_enabled",
-      :display_name => _("PCM-enabled"),
-      :description  => _("Performance and Capacity Monitoring data collection enabled"),
-      :value        => collector.pcm_enabled[sys.uuid].aggregation,
-      :read_only    => true
-    )
+    if collector.pcm_enabled[sys.uuid]
```

Comment:
`Host#pcm_enabled` defaults to false when the advanced setting isn't present so :+1: to skipping this if the host is disconnected